src/display/pattern_helper.js

```diff
@@ -360,7 +360,7 @@ var TilingPattern = (function TilingPatternClosure() {
       var graphics = canvasGraphicsFactory.createCanvasGraphics(tmpCtx);
       graphics.groupLevel = owner.groupLevel;
 
-      this.setFillAndStrokeStyleToContext(tmpCtx, paintType, color);
+      this.setFillAndStrokeStyleToContext(tmpCtx, paintType, color, graphics);
```

Comment:
let's change signature to setFillAndStrokeStyleToGraphics(graphic, paintType, color) -- `context` is accessible via graphics.ctx.